models/attachment.go

```diff
@@ -235,11 +255,13 @@ type Attachmentable interface {
     Entityable
     i878a80d2330e89d26896388a3f487eef27b0a0e6c010c493bf80be1452208f91.Parsable
     GetContentType()(*string)
+	GetContentBytes() *[]byte
```

Comment:
nit: alignment mismatch. Apply formatter here

cc: @rkodev, what's the formatting standard?